module_end_wildfire_smoke.r

```diff
@@ -152,4 +156,25 @@ endWildfireSmoke <- function(input, output, session){
       id <- showNotification("Markdown file moved to outputs/qmd directory.")
     }
   })
+  ## Download files
+  
+  output$download_report <- downloadHandler(
+    filename = function() {
+      sprintf("%s_wildfire_smoke_end.zip", Sys.Date())
+    },
+    content = function(file) {
+      # Build file paths correctly using sprintf()
+      files_to_zip <- c(
+        sprintf("outputs/qmd/%s_wildfire_smoke_end.md", Sys.Date()),
```

Comment:
Please use `file.path` to have better compatibility between Windows and Linux (shinyapps) servers due to different file separation characters.

i.e. for this one
`file.path("outputs", "qmd", sprintf("%s_wildfire_smoke_end.md", Sys.Date()))`